File: ShopBK/src/main/java/com/example/shopbk/Model/Prodotti.java
package com.example.shopbk.Model;


import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name="prodotti")
public class Prodotti {

    @Id
    @GeneratedValue(strategy= GenerationType.AUTO)
    private int id_prodotti;

    private String nome_prodotti;
    private double prezzo;
    private String img_prodotti;

}

File: ShopFR/src/app/model/prodotti.ts
export class Prodotti{

    private nome_prodotti !: string;
    private prezzo !: number;
    private img_prodotti !: string;

    constructor(nome_prodotti : string, prezzo : number, img_prodotti : string)
    {
        this.nome_prodotti = nome_prodotti;
        this.prezzo = prezzo;
        this.img_prodotti = img_prodotti;
    }

    getNome_prodotti(){ return this.nome_prodotti; }
    getPrezzo(){ return this.prezzo; }
    getImg_prodotti(){ return this.img_prodotti; }


}